src/contrib/cluster/Akka.DistributedData.Tests/Akka.DistributedData.Tests.csproj

```diff
@@ -5,6 +5,7 @@
   <PropertyGroup>
     <AssemblyTitle>Akka.DistributedData.Tests</AssemblyTitle>
     <TargetFrameworks>$(NetFrameworkTestVersion);$(NetTestVersion);$(NetCoreTestVersion)</TargetFrameworks>
+    <PlatformTarget>x64</PlatformTarget>
```

Comment:
Might want to add a comment explaining why this is necessary - feel free to add it in a new PR @Arkatufus